2016-host-organization-application/index.css

```diff
@@ -46,6 +46,10 @@
   padding: 0px 50px 25px 85px;
 }
 
+.sub-goal-points {
+  margin-left: 20px;
```

Comment:
If you check out the patch and see that 20px is what it needs to be.
On Jul 28, 2015 9:17 AM, "ScottDowne" <notifications@github.com> wrote:

> In 2016-host-organization-application/index.css
> <https://github.com/mozilla/advocacy.mozilla.org/pull/23#discussion_r35645000>
> :
>
> > @@ -46,6 +46,10 @@
> >    padding: 0px 50px 25px 85px;
> >  }
> >
> > +.sub-goal-points {
> > +  margin-left: 20px;
>
> Bullets should have a natural padding/margin thingy already. Something
> might be removing it, then we add it back. Seems a bit roundabout. Would
> the natural spacing be enough/too much? I don't think it's 20px, probably
> much more.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/advocacy.mozilla.org/pull/23/files#r35645000>.
>